tests/unittests/sources/test_ec2.py

```diff
@@ -210,6 +210,11 @@
 
 M_PATH_NET = "cloudinit.sources.DataSourceEc2.net."
 
+TAGS_METADATA_2021_03_23 = {
+    **DEFAULT_METADATA,
+    "tags": {"instance": {"Environment": "production", "Application": "test"}},
```

Comment:
Could you also please add a tag without value example here too, just so we have a reference of what's expected on Empty vals.
```suggestion
    "tags": {"instance": {"Environment": "production", "Application": "test", "TagWithoutValue": ""}},
```